### ISSUE TITLE
Add Mark Harwood's Fraud Demo

### ISSUE THREAD

**Issue by michaelkeevildown** [elastic/examples#127]
I am going to add one of @markharwood Fraud Demos into the Examples repo.

Demo Video: https://www.youtube.com/watch?v=liMhiiyQ9co

**Comment by gingerwizard** [comment]
@michaelkeevildown do you have a PR for this?

**Comment by michaelkeevildown** [comment]
it is in the works, something that is on my long list, but there will be a PR coming shortly.

**Comment by gingerwizard** [comment]
@michaelkeevildown do we still want to include this?

**Comment by michaelkeevildown** [comment]
For now I am going to close this, there were a couple of days of hype around getting this implemented from the SA team that has now died down. I am going to close this and if someone wants this demo I will get them to open a new issue and also implement it.

**Comment by markharwood** [comment]
>there were a couple of days of hype around getting this implemented from the SA team that has now died down

FYI - I have a [blog post](https://github.com/elastic/content/issues/1892) pending on tackling hate speech that references this demo video.
If website owners rely on site users to report undesirable content that is another potential source of review-fraud. In the blog I reference the same collusion-detection approaches used in this demo.
